certgen/certProcessor/src/main/java/org/incredible/certProcessor/JsonKey.java

```diff
@@ -45,4 +46,86 @@
 
     String ITEXT_LICENSE_ENABLED = "ITEXT_LICENSE_ENABLED";
     String ITEXT_LICENSE_PATH = "ITEXT_LICENSE_PATH";
+
+    String CLASS = "class";
```

Comment:
Do you think we are using such many stuff? And is that used in certProcessor package or elsewhere?

These variables are meant to be used at the Actor or service layer mostly, I would guess. Creating this global trash can of variables isn't a nice idea generally. Each package can have its own stuff. If we need to exchange, then it means the design isn't proper and requires reconsideration.

Reply:
done.